tools/docker-compose/start_tests.sh

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash
 set +x
 
-if [ `id -u` -ge 500 ] || [ -z "${CURRENT_UID}" ]; then
```

Comment:
Drive-by cleanup. This hasn't been necessary since https://github.com/ansible/awx/pull/3870 merged